.gitlab-ci.yml

```diff
@@ -60,3 +60,4 @@ production:
         K8S_SECRET_MAILER_EMAIL_BACKEND: "anymail.backends.mailgun.EmailBackend"
         K8S_SECRET_DEFAULT_FROM_EMAIL: "no-reply@hel.ninja"
         K8S_SECRET_FIELD_ENCRYPTION_KEYS: "$GL_STABLE_FIELD_ENCRYPTION_KEYS"
+        FORCE_SCRIPT_NAME: "/profiili"
```

Comment:
I'm not totally sure about this, but would we need to add additional `static-map = /profiili/static=/app/static` to `uwsgi.ini` for static files?

Reply:
@vikoivun speculated the same but then decided that we probably don't need that 👍

Reply:
I think I've been confused if I said that. As the static files seem to be the thing breaking the graphql introspection tool: https://api.hel.fi/profiili/graphql/
Django should, however, generate correct static file paths, given correct force_script_name setting.

Reply:
Maybe `MEDIA_URL` `STATIC_URL` settings need to include `FORCE_SCRIPT_NAME` as a prefix? :thinking: